ui/src/services/k8s/api.js

```diff
@@ -1,11 +1,9 @@
 import ApiClient from '../ApiClient';
 
-import {
-  Config,
-  CoreV1Api,
-  CustomObjectsApi,
-  StorageV1Api
-} from '@kubernetes/client-node';
+import { Config } from '@kubernetes/client-node/dist/config';
```

Comment:
Given this, you'll want to update the title of this PR.

Reply:
Updated :)